spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -105,7 +108,7 @@ def assert_specific_container_group
   end
 
   def assert_specific_container_node
-    @containernode = ContainerNode.first
+    @containernode = ContainerNode.where(:name => "127.0.0.1").first
```

Comment:
@abonas `X.where(x).first` == `X.find_by(x)`. So, it's just shorter -- not so much a "please change", as an "FYI, this is okay too".